server/src/main/resources/db/common/V71__create_frozen_templates_table.sql

```diff
@@ -2,7 +2,7 @@ DROP TABLE if EXISTS frozen_templates;
 
 CREATE TABLE frozen_templates (
     id varchar PRIMARY KEY,
-    creator_id varchar REFERENCES member_profile(id),
+    template_creator_id varchar REFERENCES member_profile(id),
```

Comment:
why are we renaming the column in this migration file instead of creating a new migration file with alter command ? I also see that this file is renamed from V75 to V71 so I am sure flyway will freak out unless migrations are actually not running on feedback branch ?

Reply:
I thought I could just rename stuff that hasn't been merged into 'feedback' yet. Also, I thought I could safely renumber migrations since it is something I have been asked to do before.